packages/sku/package.json

```diff
@@ -140,7 +139,6 @@
     "@types/which": "^3.0.0",
     "@vanilla-extract/css": "^1.0.0",
     "@vocab/react": "^1.0.1",
-    "assert": "^2.0.0",
```

Comment:
We should bump `babel-plugin-unassert` to latest, potentially [adding](https://github.com/unassert-js/babel-plugin-unassert/blob/master/README.md#customization) `node:assert/strict` to the list of [default patterns](https://github.com/unassert-js/babel-plugin-unassert/blob/master/index.js#L15-L24).

Reply:
Good idea.

Reply:
We wouldn't want sku to polyfill it, but configuring `node:assert/strict` in the babel plugin would allow consumers to use it a dev time.

Reply:
It needs to be polyfilled to run in the browser (`sku start`).

Reply:
Oh right. I guess the dep should be added to the fixture then.

Reply:
On master, `sku start` fails for that fixture because it can't find the `assert` package. It used to work with Yarn because of hoisting.
However, we don't test `sku start` 😅